verilog/CST/identifier.cc

```diff
@@ -58,6 +58,15 @@ const verible::SyntaxTreeLeaf* GetIdentifier(const verible::Symbol& symbol) {
   return leaf;
 }
 
+const absl::string_view GetMacroCallId(const verible::Symbol& symbol) {
+  auto t = symbol.Tag();
+  if (t.kind != SymbolKind::kNode) return nullptr;
+  if (NodeEnum(t.tag) != NodeEnum::kMacroCall) return nullptr;
+  const auto& node = down_cast<const verible::SyntaxTreeNode&>(symbol);
+  const auto* leaf = down_cast<const verible::SyntaxTreeLeaf*>(node[0].get());
```

Comment:
This can all be combined into `GetSubtreeAsLeaf`:

https://github.com/google/verible/blob/9f9a7e112105b2f69ea42a4fdf8c1e182bf1f1c8/common/text/tree_utils.h#L102

Reply:
Done